<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarClientes</class>
 <widget class="QDialog" name="AdicionarClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabCliente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>531</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabIncluirCliente">
    <attribute name="title">
     <string>Incluir</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>371</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNomeIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Endereço:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEnderecoIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTelefoneIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCPFIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIncluir">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Incluir</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPesquisarCliente">
    <attribute name="title">
     <string>Pesquisar e/ou Editar</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>361</width>
       <height>311</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id do Cliente:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIdClienteEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPesquisar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
     <widget class="QFrame" name="frPesquisarCliente">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>321</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnEdit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNomeEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Endereço:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtEnderecoEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Telefone:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTelefoneEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CPF:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCPFEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabExcluirCliente">
    <attribute name="title">
     <string>Excluir</string>
    </attribute>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>381</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="txtIdClienteExcluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluir">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dado Excluido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDadoExcluido">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>331</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id do Cliente:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabListarCliente">
    <attribute name="title">
     <string>Lista de clientes</string>
    </attribute>
    <widget class="QTableWidget" name="twCliente">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>501</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMostrarLista">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Lista</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
